### PR TITLE
Sr fix broken deploy 2

### PR DESCRIPTION
**Pull request by simen-rekkedal**
## Related Issue(s)
- #{issue number}

## Verification
- [ ] **Your** code builds clean without any errors or warnings
- [ ] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [ ] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: src/Authentication/Program.cs [hunk 1]
```diff
@@ -436,7 +436,6 @@ void ConfigurePostgreSql()
         }
 
         var connectionStringBuilder = new NpgsqlConnectionStringBuilder(postgresAdminConnectionString);
-        workspacePath = Path.Combine(Environment.CurrentDirectory, builder.Configuration.GetValue<string>("PostgreSQLSettings:WorkspacePath"));
```
Comment (acn-dgopa) [inline src/Authentication/Program.cs:439]: why is this removed?

Comment (simen-rekkedal) [reply src/Authentication/Program.cs:439]: duplicate of line 428

**Review by acn-dgopa — APPROVED** [review]
